src/api/request-validations.ts

```diff
@@ -39,3 +41,21 @@ export const validateBodyEquals = (request: RequestExt, expectedBody: unknown |
 const isBodyEmpty = (body: string | undefined | null) => {
   return body == null || body === "";
 };
+
+const coerceDateXml = (xml: string): string => {
+  return xml.replace(/(\d\d\d\d-\d\d-\d\d[Tt]\d\d:\d\d:\d\d\.\d\d\d)\d{0,4}([Zz]|[+-]00:00)/g, "$1Z");
+};
+
+// Check whether the XML request body is valid
```

Comment:
Use `/**` for javascript docs so it shows with intelisense
```suggestion
 /**
  * Check whether the XML request body is matching the given xml.
  */
```